syntax = "proto3";

package synapse;

message OpticalBroadbandOptions {
  uint32 pixel_count = 1;
  repeated uint32 bit_width = 2;
  repeated uint32 sample_rate = 3;
  repeated uint32 gain = 4;
}

message OpticalBroadbandConfig {
  repeated bool pixel_mask = 1;
  uint32 bit_width = 2;
  uint32 sample_rate = 3;
  uint32 gain = 4;
}
